CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.1.2
+
+### Bug fixes
+
+* Kotlin projects no longer creates the `RealmDefaultModule` if no Realm model classes are present (#3746).
```

Comment:
Yes, it only effects Kotlin projects. It was still a bug in the annotation processor code, but it was only triggered by Kotlin.